src/audioclient.rs

```diff
@@ -2,6 +2,8 @@
 // Licensed under the MIT License <LICENSE.md>
 //! this ALWAYS GENERATED file contains the definitions for the interfaces
 //1627
+use unknwnbase::*;
```

Comment:
If I try to do `::IUnknown` the RIDL macro doesn't compile, so I need to import the type manually. Would it be better to change the macro so that it supports absolute paths?